docker/Makefile

```diff
@@ -14,13 +14,13 @@ create-baseimg: prepare-docker-buildx
 	docker buildx build -t $(BASE_IMAGE) --push \
 		--build-arg root_image=$(ROOT_IMAGE) \
 		--build-arg cert_image=$(CERT_IMAGE) \
-		--platform=$(PLATFORMS) \
+		--platform=linux/amd64 \
```

Comment:
again, if you need to change the behavior, just override the PLATFORMS var, don't change the actual target code

Reply:
Yes, doing that right now.